api_tests/src/user.spec.ts

```diff
@@ -130,7 +130,11 @@ test("Requests with invalid auth should be treated as unauthenticated", async ()
 });
 
 test("Create user with Arabic name", async () => {
-  let user = await registerUser(alpha, alphaUrl, "تجريب");
+  let user = await registerUser(
+    alpha,
+    alphaUrl,
+    "تجريب" + Math.random().toString().slice(2, 5),
```

Comment:
i don't like this since it's fairly easy to call this function twice in the same millisecond which would then cause an issue . math.random() should give at least ~60bits of randomness much less chance of collision. the slice is a bit ugly i agree (needed due to username restirctions) but i think it's fine.